filecoin-proofs/src/bin/parampublish.rs

```diff
@@ -47,31 +47,30 @@ Set $FILECOIN_PARAMETER_CACHE to specify parameter directory. Defaults to '{}'
 
     if let Some(matches) = matches.subcommand_matches("publish") {
         let mut new_parameter_map: ParameterMap = HashMap::new();
-        let parameters = if matches.is_present("all") {
+        let parameter_ids = if matches.is_present("all") {
             get_local_parameter_ids()
         } else {
             choose_local_parameter_ids()
         }
         .expect(ERROR_PARAMETERS_LOCAL);
 
-        if parameters.len() > 0 {
+        if parameter_ids.len() > 0 {
             println!("publishing parameters");
 
-            for parameter in parameters.iter() {
-                println!("publishing '{}'...", parameter);
+            for parameter_id in parameter_ids.iter() {
```

Comment:
if ypu use into_iter you get an owned string so you can use that to insert into the hashmap below